.pre-commit-config.yaml

```diff
@@ -0,0 +1,15 @@
+# See https://pre-commit.com for more information
```

Comment:
I'm not sure if I'm coming too late with the unpopular opinion here 🙃 I'm not super fond of using autoformatters.

Reply:
I really like them (`pyupgrade` and `autoflake` are ones I also like using), I think we should certainly have a conversation about it. I would say that `black` tends to make math worse, but we can also tell it to skip those lines.

Reply:
We'll settle this argument elsewhere :)